services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/table/TableQueryManagerImplTest.java

```diff
@@ -1247,6 +1261,29 @@ public void testRunFacetQueries(){
 		verifyNoMoreInteractions(mockTableIndexDAO, mockFacetModel,mockTransformer1, mockTransformer2);
 
 	}
+	
+	private RowSet createRowSetForTest(List<String> headerNames, List<String>... rowValues){
+		RowSet rowSet = new RowSet();
+		List<SelectColumn> headerObjects = new ArrayList<>();
+		
+		//select column for first row

```

Comment:
Forgot to remove that comment when i copypasted and modified the Rowset constructor